AgContext.yaml

```diff
@@ -0,0 +1,269 @@
+      - lupins
+      - chickpeas
+      - lentils
+      - field_peas
+      - faba_beans
+      - mung_beans
+      - canola
+      - mustard
+      - flax
+      - linseed
+      - safflower
+      - cotton
+      - carrots
+      - weed_only
+      - other
+    description: |-
+          Crop type.
+          One of several strings describing the crop grown in the image.
+
+  grains_descriptive_text:
+    type:
+      string
+    enum:
+      - emergence
+      - seedling
+      - tillering
+      - stem_elongation
+      - booting
+      - ear_emergence
+      - flowering
+      - milky_dough
+      - dough
+      - ripening
+      - na
+    description: |-
+        Grains descriptive text.
+        One of ten possible strings describing the crop developmental stage.
+        If this AgContext is not in a cropping environment, use value "na".
+
+  bbch_descriptive_text:
+    type:
+      string
+    enum:
+      - germination
+      - sprouting
+      - bud development
+      - leaf development
+      - formation of side shoots
+      - tillering
+      - stem elongation
+      - rosette growth
+      - shoot development
+      - development of harvestable vegetative parts
+      - bolting
+      - inflorescence emergence
+      - heading
+      - flowering
+      - development of fruit
+      - ripening or maturity of fruit and seed
+      - senescence
+      - beginning of dormancy
+      - na
+    description: |-
+        BBCG descriptive text.
+        One of several possible strings describing the stage of the crop, chosen from a list of possible terms used by the BBCH.
+        If this AgContext is not in a cropping environment, use value "na".
+
+  bbch_code:
+    type:
+      string
+    pattern: "^gs[0-9][0-9]$|^na$|^(?!.*gs00))"
+    description: |-
+        BBCG descriptive text.
+        One of several possible strings describing the stage of the crop, chosen from a list of codes used by the BBCH.
+        If this AgContext is not in a cropping environment, use value "na".
+
+  soil_colour:
+    type:
+      string
+    enum:
+      - not_visible
+      - black
+      - dark_brown
+      - brown
+      - red_brown
+      - dark_red
+      - yellow
+      - pale_yellow
+      - white
+      - grey
+    description: |-
+        Soil colour.
+        General description of the soil colour.
+
+  surface_cover:
+    type: string
+    enum:
+      - cereal
+      - oilseed
+      - legume
+      - cotton
+      - black_plastic
+      - white_plastic
+      - woodchips
+      - other
+      - none
+    description: |-
+      Surface cover type.
+      One of several strings describing the background cover that is behind any plants in the images.
+
+  surface_coverage:
+    type: string
+    enum:
+      - "0-25"
+      - "25-50"
+      - "50-75"
+      - "75-100"
+    description: |-
+      Percent of coverage in image.
+      Approximate measurement of the percent of the soil in the image that is covered by the surface cover.
```

Comment:
I agree that this sounds like it may be of questionable utility; I had clarified this with Guy and it is intended to be a rough estimate for the entire dataset.